study.md

```diff
@@ -85,7 +93,7 @@ Using Hash methods and string interpolation in Ruby, create a string using
 conspiracy theories.".
 
 ```ruby
-# your answer here
+p `my name is #{dale[:given_name]} #{dale[:family_name]} and i'm an #{dale[:occupation]} that enjoys #{dale[:hobbies][1]}.`
```

Comment:
If the grave accents were double quotation marks, then this will work correctly.